src/main/java/com/google/jenkins/plugins/computeengine/ComputeEngineWindowsLauncher.java

```
@@ -125,14 +115,15 @@ private boolean bootstrap(ComputeEngineComputer computer, TaskListener listener)
       }
       if (!isAuthenticated) {
         logWarning(computer, listener, "Authentication failed");
-        return false;
+        return Optional.empty();
       }
-    } finally {
+    } catch (Exception e) {
```

Same comment as above re: swallowing exceptions.

---

Added logging